lib/model/model.go

```diff
@@ -1748,18 +1748,6 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 	m.fmut.Unlock()
 	mtimefs := fset.MtimeFS()
 
-	// Check if the ignore patterns changed as part of scanning this folder.
-	// If they did we should schedule a pull of the folder so that we
-	// request things we might have suddenly become unignored and so on.
-
-	oldHash := ignores.Hash()
-	defer func() {
-		if ignores.Hash() != oldHash {
-			l.Debugln("Folder", folder, "ignore patterns changed; triggering puller")
-			runner.IndexUpdated()
-		}
-	}()
-
```

Comment:
How is this compensated?

Reply:
By https://github.com/syncthing/syncthing/pull/4340/files/d06035c89cab60277cf849c03aff2ddf2395d754#diff-1df9805b2381b6288b220226f82d5dc9R1771 and then https://github.com/syncthing/syncthing/pull/4340/files/d06035c89cab60277cf849c03aff2ddf2395d754#diff-feb9fde15f8dd977fe9b279a36a447a2R163